manifest-generator/generator-services/src/main/java/io/hyscale/generator/services/processor/PluginProcessor.java

```diff
@@ -119,7 +118,14 @@
                 WorkflowLogger.endActivity(Status.FAILED);
             }
         });
-        warningMessageForCustomSnippets(kindVsCustomSnippets);
+        ServiceMetadata serviceMetadata = new ServiceMetadata();
+        serviceMetadata.setAppName(manifestContext.getAppName());
+        serviceMetadata.setEnvName(manifestContext.getEnvName());
+        serviceMetadata.setServiceName(serviceName);
+        List<Manifest> customSnippetsManifestList = getManifestsFromCustomSnippets(serviceMetadata, manifestDir, kindVsCustomSnippets);
```

Comment:
better to write all the generated resources in manifestDir at once instead of duplicating code